modules/zeta_global_sspBidAdapter.js

```diff
@@ -178,6 +217,49 @@ function buildBanner(request) {
   };
 }
 
+function buildVideo(request) {
+  let video = {};
+  const videoParams = utils.deepAccess(request, 'mediaTypes.video', {});
+  for (const key in VIDEO_CUSTOM_PARAMS) {
+    if (videoParams.hasOwnProperty(key)) {
+      video[key] = checkParamDataType(key, videoParams[key], VIDEO_CUSTOM_PARAMS[key]);
+    }
+  }
+  if (videoParams.playerSize) {
+    if (utils.isArray(videoParams.playerSize[0])) {
+      video.w = parseInt(videoParams.playerSize[0][0], 10);
+      video.h = parseInt(videoParams.playerSize[0][1], 10);
+    } else if (utils.isNumber(videoParams.playerSize[0])) {
+      video.w = parseInt(videoParams.playerSize[0], 10);
```

Comment:
Why does `parseInt` need to be run on a value that is already a number?

Reply:
@msm0504 **45.5** it's a number, but we need only _int_ values - **45**.